railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -547,6 +547,28 @@ def run_after_bundle_callbacks
         @after_bundle_callbacks.each(&:call)
       end
 
+      def show_next_steps_banner
+        return if options[:quiet]
+
+        puts <<~NEXTSTEP
+
+Rails project created and dependencies installed.
+
+Now switch to the directory where the Rails app was created:
+
+  $ cd #{app_path}
+
+Then boot the development server:
+
+  $ ./bin/rails server
+
+You can always get help by running:
+
+  $ ./bin/rails help
+
+        NEXTSTEP
```

Comment:
The advantage of the squiggly HEREDOC is that indentation of the least-indented line will be
removed from each line of the content:

```suggestion
        puts <<~NEXTSTEP

          Rails project created and dependencies installed.
          
          Now switch to the directory where the Rails app was created:

            $ cd #{app_path}

          Then boot the development server:

            $ ./bin/rails server

          You can always get help by running:

            $ ./bin/rails help
          
        NEXTSTEP
```

Reply:
Changes pushed